docs/install-and-run.md

```diff
@@ -1,208 +1,239 @@
+        configure_for_earlgrey(installer, PROJECT_NAME, TEST_TARGET, SCHEME_FILE) # EarlGrey Gem Call
+      end
+      ```
 
-After you successfully run the `pod install` command, open the generated workspace and find EarlGrey
-installed in the `Pods/` directory. The generated `Pods/` project should look similar to:
+      Once finished, do a `pod install`. For compatibility between different versions, please see [this doc](https://github.com/google/EarlGrey/tree/master/docs/versions.md). To download a particular version of the gem, use `gem install earlgrey -v x.y.z`.
 
-  <img src="images/image02.png" width="250">
+#### Step 3: Open the workspace and verify that you see EarlGrey.
 
-### Carthage Installation
+   After you successfully run the `pod install` command, open the generated workspace and find EarlGrey
+   installed in the `Pods/` directory. The generated `Pods/` project should look similar to:
+
+   <img src="images/image02.png" width="250">
 
-EarlGrey supports Carthage via [the EarlGrey gem.](https://rubygems.org/gems/earlgrey)
+### Carthage Installation
 
 #### Step 1: Set up a test target for Carthage
 
-See [Step 1 from the `CocoaPods installation`](#step-1-set-up-a-test-target) detailed above.
+   See [Step 1 from the `CocoaPods installation`](#step-1:-set-up-a-unit-target) detailed above.
 
 #### Step 2: Configure carthage
 
-Install Carthage
+   1. Install Carthage
 
-`brew update; brew install carthage`
+      `brew update; brew install carthage`
 
-Specify the version of EarlGrey to use in Cartfile.private.
-Note that you can also use "master" instead of a release tag.
+   2. Specify the version of EarlGrey to use in Cartfile.private.
 
-`echo 'github "google/EarlGrey" "1.2.0"' >> Cartfile.private`
+      Note that you can also use "master" instead of a release tag.
 
-Update to latest EarlGrey revision and create Cartfile.resolved.
+      `echo 'github "google/EarlGrey" "1.2.0"' >> Cartfile.private`
 
-`carthage update EarlGrey --platform ios`
+   3. Update to latest EarlGrey revision and create Cartfile.resolved.
+
+      `carthage update EarlGrey --platform ios`
 
 #### Step 3: Use the EarlGrey gem
 
-Install the EarlGrey gem.
+   1. Install the EarlGrey gem.
 
-`gem install earlgrey`
+      `gem install earlgrey`
 
-Use the gem to install EarlGrey into the testing target.
+   2. Use the gem to install EarlGrey into the testing target.
 
-`earlgrey install -t EarlGreyExampleSwiftTests`
+      `earlgrey install -t EarlGreyExampleSwiftTests`
 
-Now you're ready to start testing with EarlGrey!
-If you need more control, review the available install options.
+      Now you're ready to start testing with EarlGrey!
+      If you need more control, review the available installation options in the `earlgrey gem`:
 
-`earlgrey help install`
+      `earlgrey help install`
 
 ### GitHub Installation
 
 In cases where CocoaPods is not compatible with your project, you can add EarlGrey manually to your Xcode project.
```

Comment:
Don't need to mention cocoapods not being compatible...Just say some like: if cocoapods or carthage don't work you then you can integrate manually.